test/timezone_test.exs

```diff
@@ -127,4 +127,25 @@ defmodule TimezoneTests do
 
     assert 63645015540 = DateTime.to_seconds(Timex.datetime(datetime, "Europe/Vienna"), :zero, [utc: true])
   end
+
+  test "name_of" do
+    assert {:error, {:invalid_timezone, :wat}} = Timezone.name_of(:wat)
+    assert {:error, {:invalid_timezone, "America/Not_A_Real_Timezone"}} = Timezone.name_of("America/Not_A_Real_Timezone")
+    assert "America/Los_Angeles" = Timezone.name_of("America/Los_Angeles")
+    assert "Etc/GMT+0500" = Timezone.name_of("GMT+0500")
```

Comment:
It's important to note that the etcetera zones have a specific format (non zero-padded), and that none of them support seconds granularity. If as I suggested we fake them, we should adhere to the convention, so this zone would be `Etc/GMT+5:45`. I want to be clear about the differentiation between hours and seconds, hence the colon.